preferencesfx/src/main/java/com/dlsc/preferencesfx/util/AbstractStorageHandler.java

```diff
@@ -0,0 +1,148 @@
+package com.dlsc.preferencesfx.util;
+
+import com.dlsc.preferencesfx.model.Setting;
+import javafx.collections.ObservableList;
+import java.util.prefs.Preferences;
+
+/**
+ * Abstraction for storing values of {@link Setting}.
+ * Clients able to provide custom implementations by extending this class.
+ *
+ * @author François Martin
+ * @author Marco Sanfratello
+ * @author kivimango
+ */
+
+public abstract class AbstractStorageHandler {
```

Comment:
What's the rationale to make this an abstract `class` rather than keep it as an `interface`? There's no explicit behavior provided by this type, only a contracr, thus an `interface` would suffice.

Reply:
Hello ! My first thought was to use an interface of course, but i think some behavior could be separated and moved into the AbstractSotrageHandler in the future (like loadWindowWidth() etc. methods related to the GUi, and keep loabObject() and saveObject() abstract, since its business logic-related.)
But i can refactor to use interface if you want.